docs/1.2/core/reference/cored.md

```diff
@@ -83,6 +83,13 @@ the limit will receive an HTTP 429 response.
 
     Can be stacked with **RATELIMIT_TOKEN**.
 
+* **BOOTURL**: Setting this value causes the `cored` process to join an
```

Comment:
In Core 1.2, to join an existing a cluster you provided this environment variable. In d20cecb we removed this environment variable, replacing it with the `corectl join` command.